lib/index.js

```diff
@@ -197,38 +201,85 @@ const addToNameCal = (food) => {
 }
 
 const prependToBreakfast = (food) => {
-  $('.breakfasts').prepend(` 
+  $('.breakfasts').prepend(`
     <article class='breakfast' data-breakfast-food-id=${food.id}>` + addToNameCal(food) +
       `<p class='delete'><button data-breakfast-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
     </article>`
   )
 }
 
 const prependToLunch = (food) => {
-  $('.lunches').prepend(` 
+  $('.lunches').prepend(`
     <article class='lunch' data-lunch-food-id=${food.id}>` + addToNameCal(food) +
       `<p class='delete'><button data-lunch-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
     </article>`
   )
 }
 
 const prependToDinner = (food) => {
-    $('.dinners').prepend(` 
+    $('.dinners').prepend(`
       <article class='dinner' data-dinner-food-id=${food.id}>` + addToNameCal(food) +
         `<p class='delete'><button data-dinner-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
       </article>`
     )
 }
 
 const prependToSnack = (food) => {
-  $('.snacks').prepend(` 
-    <article class='snack' data-snack-food-id=${food.id}>` + addToNameCal(food) + 
+  $('.snacks').prepend(`
+    <article class='snack' data-snack-food-id=${food.id}>` + addToNameCal(food) +
       `<p class='delete'><button data-snack-food-id=${food.id} class='delete-food-btn'><i class="fa fa-trash"></i></button></p>
     </article>`
   )
 }
 
+const getBreakfastCalorieSum = (meal) => {
+  var total = 0
+  var sum = meal.foods.forEach(function(food) {
+    total += food.calories
+  })
+  appendToBreakfastCalories(total)
+}
+
+const appendToBreakfastCalories = (total) => {
+  $('.breakfast-calorie-sum').append(total);
+}
+
+const getLunchCalorieSum = (meal) => {
+  var total = 0
+  var sum = meal.foods.forEach(function(food) {
+    total += food.calories
+  })
+  appendToLunchCalories(total)
+}
+
+const appendToLunchCalories = (total) => {
+  $('.lunch-calorie-sum').append(total);
+}
+
+const getDinnerCalorieSum = (meal) => {
+  var total = 0
+  var sum = meal.foods.forEach(function(food) {
+    total += food.calories
+  })
+  appendToDinnerCalories(total)
+}
+
+const appendToDinnerCalories = (total) => {
+  $('.dinner-calorie-sum').append(total);
+}
```

Comment:
I wonder if we can somehow interpolate these and just have one method instead of 4?

Reply:
I'm actually going to try this now, I'll let you know if I get anywhere with it!